djhubspot/events.py

```diff
@@ -39,23 +30,46 @@ class HubspotEvent:
         'company.propertyChange': EVENT_TYPE_CONTACT_UPDATED,
         'deal.creation': EVENT_TYPE_DEAL_CREATED,
         'deal.deletion': EVENT_TYPE_DEAL_DELETED,
-        'deal.propertyChange' EVENT_TYPE_DEAL_UPDATED,
+        'deal.propertyChange': EVENT_TYPE_DEAL_UPDATED,
     }
 
     message = None
     event_type = None
 
     def __init__(self, message, **kwargs):
+        """
+
+        Parameters
+        ----------
+        message: dict
+        """
         self.message = message
 
-        raw_event = self.message.get('event')
+        raw_event_type = self.message.get('subscriptionType')
         try:
-            self.event_type = self.MESSAGE_EVENT_TO_EVENT_TYPE[raw_event]
+            self.event_type = self.MESSAGE_EVENT_TO_EVENT_TYPE[raw_event_type]
         except KeyError:
-            logger.warning("Unrecognized Hubspot event type: ", raw_event)
+            logger.warning("Unrecognized Hubspot event type: ", raw_event_type)
+            raise HubspotEventError()
         else:
             self.parse_event()
 
+    # Common event properties
+    # ------------------------------------------------------------------------------
+
+    @property
+    def occurred_at(self):
+        """
+        When this event occurred.
+
+        Returns
+        -------
+        datetime
+        """
+        ms = self.message['occurredAt']
```

Comment:
Are we sure all HS events have this property? Maybe `self.message.get('occurredAt')` would be safer?

Reply:
Yes, it is a kind of common property. I did not think two much about it yet.

I think that we should reject the events that does not contains expected keys. Or maybe expose a property `is_valid` and let the user handle this?